omnibus/config/patches/snowflake-connector-python-py2/dependencies.patch

```diff
@@ -1,9 +1,13 @@
 diff --git a/setup.py b/setup.py
-index 0da6a8a..761de39 100644
+index 0da6a8a..509e3a1 100644
 --- a/setup.py
 +++ b/setup.py
-@@ -185,13 +185,13 @@ setup(
-         'certifi<2021.0.0',
+@@ -182,16 +182,16 @@ setup(
+         'botocore>=1.5.0,<1.14.0',
+         'requests<2.23.0',
+         'urllib3>=1.20,<1.26.0',
```

Comment:
these constraints are new, are they intentional?

Reply:
They're not, they've been added because `git diff` always shows 3 lines before the first changed line, and the first changed line now happens earlier (https://github.com/snowflakedb/snowflake-connector-python/blob/aefe584e1d70f30d01ee7304abd520a2089cde58/setup.py#L182-L184).

Reply:
Oh, thanks for clarifying! Got confused by github's color scheme on PR diffs of a patch file :)